### PR TITLE
Add productId and priceId to Camp model

### PR DESCRIPTION
**Pull request by jjjding2**
Small model change

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by annechung — APPROVED** [review]

File: backend/typescript/models/campSession.model.ts [hunk 1]
```diff
@@ -28,31 +33,30 @@ const CampSessionSchema: Schema = new Schema({
...
     type: [Date],
     required: true,
   },
-  startTime: {
+  endTime: {
     type: String,
     required: true,
   },
-  endTime: {
+  priceId: {
```
Comment (SinghHarman286) [inline backend/typescript/models/campSession.model.ts:44]: I was thinking that in `priceId`, we have `required` as false and have `default` as "". The reason being is that price Ids will be created only once `campSessions` are successfully created. So having them as required, we'll need to pass an empty string every time we are creating a campSession

File: backend/typescript/models/camp.model.ts [hunk 2]
```diff
@@ -7,13 +7,14 @@ export interface Camp extends Document {
...
   description: string;
-  location: string;
   fee: number;
-  formQuestions: (Schema.Types.ObjectId | FormQuestion)[];
-  campSessions: (Schema.Types.ObjectId | CampSession)[];
   fileName?: string;
+  formQuestions: (Schema.Types.ObjectId | FormQuestion)[];
+  location: string;
+  name: string;
+  productId: string;
```
Comment (SinghHarman286) [inline backend/typescript/models/camp.model.ts:17]: Do we need productId in camp model if we are following `prod_<mongoose_id>` syntax for stripe products?

**Review by SinghHarman286 — COMMENTED** [review]
Looks good! Just left some comments/questions I have about some of the changes

File: backend/typescript/models/camp.model.ts (hunk 2 above)
Comment (jjjding2) [reply backend/typescript/models/camp.model.ts:17]: Decided to let Stripe generate the id stored in productId

**Pull request merged**